event-types.json

```diff
@@ -1027,6 +1063,15 @@
       },
       "description": "The absorbed dose of ionizing radiation imparted at a given location per unit of time (second, minute, hour, or day). "
     },
+    "acceleration": {
+      "formats": {
+        "m-s2": {
+          "description": "Meters per second squared",
+          "type": "number"
+        }
+      },
+      "description": "The rate of change of linear velocity."
```

Comment:
Field order here as well

Reply:
@sgoumaz Regarding the order of `description` and `formats`, it's really random throughout the file. Since it's a JSON object where item fields orders have no importance and usually the client decides of this, I will it as is.

Reply:
From around [line 1000](https://github.com/pryv/data-types/blob/master/event-types.json#L1003) going down, you will find many types with description after formats.